src/guide/1/dbsearch/javascript/index.md

```diff
@@ -8,166 +8,66 @@ title: Javascript
 
 For this example we will use Node.js. You will need to install Node.js and NPM.
 
-Let's create a new project folder called `databaseSearch`:
+Let's create a new project folder called `database-search`:
 
 
 ```bash
-    mkdir databaseSearch
+    mkdir database-search
 ```
 
-Now in you `databaseSearch` folder initialize the Node.js App:
+Now install Kuzzle SDK JS 6:
 
 
 ```bash
-    npm init
+    npm install kuzzle-sdk
```

Comment:
You need to do `npm init` prior to installing package otherwise, `npm` will try to find a `package.json` in any parent folder  and add the dependence in it...